### PR TITLE
ServiceNow v2 - added support for get modified remote data

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

fixes https://github.com/demisto/etc/issues/33694

## Description
added support for get modified remote data

## Minimum version of Demisto
- [x] 6.1.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by anara123 — APPROVED** [review]
looks good

**Comment by ShirleyDenkberg** [comment]
@anara123 Doc review completed.

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 1]
```diff
@@ -79,10 +79,12 @@ If MFA is enabled for your user, follow the next steps:
 | timestamp_field | Timestamp field to filter by \(e.g., \`opened\_at\`\) This is how the filter is applied to the query: "ORDERBYopened\_at^opened\_at&gt;\[Last Run\]". To prevent duplicate incidents, this field is mandatory for fetching incidents. | False |
 | incidentType | Incident type | False |
 | get_attachments | Get incident attachments | False |
-| mirror_direction | Chose whenever to mirror the incident. You can mirror only In (from ServiceNow to XSOAR), only out(from XSOAR to ServiceNow) or both direction. | None |
-| comment_tag | Choose the tag to add to an entry to mirror it as a comment in ServiceNow. | comments |
-| work_notes_tag | Choose the tag to add to an entry to mirror it as a work note in ServiceNow. | work_notes |
-| file_tag | Choose the tag to add to an entry to mirror it as a file in ServiceNow. | ForServiceNow |
+| mirror_direction | Choose whenever to mirror the incident. You can mirror only In (from ServiceNow to XSOAR), only out(from XSOAR to ServiceNow) or both direction. | False |
```
Comment (yaakovi) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:82]:
```suggestion
| mirror_direction | Choose whenever to mirror the incident. You can mirror only In (from ServiceNow to XSOAR), only out (from XSOAR to ServiceNow), or both directions. | False |
```

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 2]
```diff
@@ -162,8 +164,12 @@ match.
 ![image](https://raw.githubusercontent.com/demisto/content/d9bd0725e4bce1d68b949e66dcdd8f42931b1a88/Packs/ServiceNow/Integrations/ServiceNowv2/doc_files/ticket-example.png)
 
 
-* The final **source of truth** for the incident for Cortex XSOAR are the **values in Cortex XSOAR**. 
-Meaning, if you change the severity in Cortex XSOAR and then change it back in ServiceNow, the final value that will be presented is the one in Cortex XSOAR.
+**Notes**
+- The final 'source of truth' for the incident for Cortex XSOAR are the values in Cortex XSOAR. 
+  Meaning, if you change the severity in Cortex XSOAR and then change it back in ServiceNow, the final value that will be presented is the one in Cortex XSOAR.
+- The integration queries ServiceNow for modified records based on the timestamp field set in the *update_timestamp_field* integration parameter and the limit set in the *mirror_limit* integration parameter.
+      If more records are modified in the timeframe when they are queried than are configured in the *limit* parameter, the extra records won't be mirrored in and the incidents in Cortex XSOAR will not be updated.
```
Comment (yaakovi) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:171]: they will not be mirrored at all, right? (unlike fetch-incident which continues to fetch from the same spot).

**Review by yaakovi — APPROVED** [review]
nice

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md (hunk 2 above)
Comment (Itay4) [reply Packs/ServiceNow/Integrations/ServiceNowv2/README.md:171]: correct

**Pull request merged**